docs/source/guides/VTFModding.rst

```diff
@@ -4,24 +4,72 @@ VTF Modding
+Since vtf modding was majorly for introducing custom weapon skins. Most of the development on VTF modding was designed focused on custom weapon skins. Although understanding how custom weapon VTF skins is specific, it is needed in order to VTF mod other textures. 
+
+Most textures in the game use .dds but we can make them use .vtf. 
+
+What we'll be doing is extracting the model. Replacing the texture path in the mdl to lead to our texture. Then using our texture and allowing for extra properties.
+
+.. _VPK Tool: https://github.com/Wanty5883/Titanfall2/blob/master/tools/Titanfall_VPKTool3.4_Portable.zip
+
+VPK Tool
+--------
+.. _cra0 VPKTool: https://github.com/Wanty5883/Titanfall2/blob/master/tools/Titanfall_VPKTool3.4_Portable.zip
+
+.. _Harmony VPKTool: https://github.com/harmonytf/HarmonyVPKTool
+
+You have 2 options for a VPK tool. Pick either the older VPK tool: `cra0 VPKTool`_ or the Newer VPK tool: `Harmony VPKTool`_ (better).
+
+With your VPK tool opened. 'Open' ``englishclient_mp_common.pak000_dir.vpk`` which is located in ``Titanfall2/vpk``. 
+
+Inside of the VPK. Not all guns are named the same in files as they are in game. Here is `list of weapon names <https://noskill.gitbook.io/titanfall2/documentation/file-location/weapon/weapon-model>`_ to help you out. 
+
+Navigate to ``models/weapons/car101``. Extract all the viewmodel versions (ptpov) and normal model (w) mdl's.
+
+Hex Editor
+----------
+
+To change the path in the .mdl to the custom .vmt. We need a hex editor. I will use `HxD`_, but you can also use `ida`_, or anything else; its personal preference. 
+
+.. _HxD: https://mh-nexus.de/en/hxd/
+.. _ida: https://hex-rays.com/ida-free/
+
+
+In your hex editor. Open your .mdl. We need to get closer to the string we need or else you'll be scrolling and searching for hours. Search:(CTRL+F) for skin_31. If you don't get any matches, try skn_31, skin31, elite, or prime.
+
+The path should look something like ``.models\Weapons_R2\weaponname\weaponname_skin_31``. Before this skin_31 path. We should find an almost exact path but without ``skin_31``. This is the path we want. This non-skin_31 path is the path to the default weapon skin.
```

Comment:
Theres a bunch of grammar mistakes here, and im not quite sure what its trying to say. which path is meant? the one shown or one without the skin31 part?

Reply:
I thought it was clear and adds addition insight. I'll ask other to see if it's also unclear to them. Then consider changes.

Reply:
Open the .mdl in a hex editor. We need to get closer to the string we need, otherwise you'll be scrolling and searching for hours. Search (CTRL+F) for skin_31. If there are no matches, try: skn_31, skin31, elite or prime.